connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedHerder.java

```diff
@@ -886,14 +898,24 @@ public void onRevoked(String leader, Collection<String> connectors, Collection<C
                     // this worker. Instead, we can let them continue to run but buffer any update requests (which should be
                     // rare anyway). This would avoid a steady stream of start/stop, which probably also includes lots of
                     // unnecessary repeated connections to the source/sink system.
-                    for (String connectorName : connectors)
-                        worker.stopConnector(connectorName);
+                    for (String connectorName : connectors) {
+                        try {
+                            worker.stopConnector(connectorName);
+                        } catch (ConnectException e) {
+                            log.error("failed to stop connector " + connectorName);
```

Comment:
Same question here about failing to stop connectors; same nit about capitalization.